Rakefile

```diff
@@ -54,6 +54,11 @@ namespace :shipyard do
     end
   end
 
+  desc 'Publish project to npm as scoped package @codeship/shipyard'
+  task :console do
+    sh 'npm publish --access public'
```

Comment:
Imo that needs to be the last part of the rakefile.

Reply:
Updated.